### PR TITLE
docs: add iOS agent links

### PR DESCRIPTION
**Pull request by bmorelli25**
This PR updates the following documentation (click to see it!)

#### Observability Guide
- [x] [Instrument apps](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/observability/master/instrument-apps.html) 

#### Fleet User Guide
- [x] [Quick start](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/fleet/master/fleet-quick-start-traces.html)

#### APM Overview
- [x] [Components](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/components.html)
- [x] [Quick start](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/install-and-run.html)
- [x] [Dropped spans](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/transaction-spans.html) Added a _not yet supported_ note
- [x] [Distributed tracing:](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/distributed-tracing.html) Tabs have been added but require input from @bryce-b for content 
- [x] [Agent/Server compatibility](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/agent-server-compatibility.html)
- [x] [Troubleshooting](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/troubleshooting.html)
~- [ ] [Metrics](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/metrics.html) Not all agents have metrics documentation. Is this something we should add for 7.14?~
- [x] [OpenTracing Bridge](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/opentracing.html)
    > I believe the otel swift sdk has open tracing support that we can point to
    * I wasn't able to find anything about this other than a quick note in the project readme that says a shim exists. Can you shed more light on this?

#### APM Server Reference
- [x] [Set up](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/server/master/next-steps.html)
~- [ ] [Server cert/pinning](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/server/master/ssl-setup.html)~
   > not sure if this is relevant to mobile...
  * Me neither...
~- [ ] [API keys](https://www.elastic.co/guide/en/apm/server/current/api-key.html)~
    > not sure if this is relevant to mobile...
   * Me neither...
- [x] [Secret token and serverURLs](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/server/master/secret-token.html) secret token added
- [x] [Troubleshooting](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/server/master/troubleshooting.html)

### CODE REVIEW

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: docs/tab-widgets/configure-agent.asciidoc [hunk 1]
```diff
@@ -11,6 +11,7 @@ for more information and a configuration reference.
 For a full list of agent configuration options, see the relevant agent reference:
 
 * {apm-go-ref-v}/configuration.html[Go Agent configuration]
+* {apm-ios-ref-v}/configuration.html[iOS Agent configuration]
```
Comment (bmorelli25) [inline docs/tab-widgets/configure-agent.asciidoc:14]: Broken in preview but fixed in https://github.com/elastic/observability-docs/commit/cc8dc2a89374207f6446c2afaa4ad35cb15c6361

**Review by bryce-b — APPROVED** [review]

**Comment by bmorelli25** [comment]
@bryce-b -- did you see my questions in the description?

**Comment by bryce-b** [comment]
@bmorelli25 I'll have to double check, but I believe it does have Open Tracing support.

**Comment by bryce-b** [comment]
yes, there is an open tracing shim : https://github.com/open-telemetry/opentelemetry-swift/tree/main/Sources/Importers/OpenTracingShim

File: docs/tab-widgets/distributed-trace-receive.asciidoc [hunk 2]
```diff
@@ -33,6 +33,17 @@ transaction := apm.DefaultTracer.StartTransactionOptions("GET /", "request", opt
 // ***************************************************
 // ***************************************************
 
+// tag::ios[]
+
+experimental::[]
+
+_coming soon_
+
+// end::ios[]
```
Comment (bmorelli25) [inline docs/tab-widgets/distributed-trace-receive.asciidoc:42]: @bryce-b do you want to fill these out in this PR or in a follow-up? [Link](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/distributed-tracing.html#_add_the_traceparent_header_to_outgoing_requests).

File: docs/tab-widgets/distributed-trace-send.asciidoc [hunk 3]
```diff
@@ -27,6 +27,17 @@ tracestate := traceContext.State.String()
...
+// tag::ios[]
+
+experimental::[]
+
+_coming soon_
+
+// end::ios[]
+
+// ***************************************************
+// ***************************************************
```
Comment (bmorelli25) [inline docs/tab-widgets/distributed-trace-send.asciidoc:39]: > @bryce-b do you want to fill these out in this PR or in a follow-up? [Link](https://apm-server_5634.docs-preview.app.elstc.co/guide/en/apm/get-started/master/distributed-tracing.html#_add_the_traceparent_header_to_outgoing_requests).

**Comment by bmorelli25** [comment]
* Added shim link, thanks
 * Crossed out metrics, server/cert pinning, and API keys
 
Just one more Q around distributed tracing which I've commented above

File: docs/tab-widgets/distributed-trace-send.asciidoc (hunk 3 above)
Comment (bryce-b) [reply docs/tab-widgets/distributed-trace-send.asciidoc:39]: 👀

File: docs/tab-widgets/distributed-trace-receive.asciidoc (hunk 2 above)
Comment (bryce-b) [reply docs/tab-widgets/distributed-trace-receive.asciidoc:42]: incoming requests isn't really applicable to mobile.

File: docs/tab-widgets/distributed-trace-receive.asciidoc [hunk 4]
```diff
@@ -33,6 +33,17 @@ transaction := apm.DefaultTracer.StartTransactionOptions("GET /", "request", opt
 // ***************************************************
 // ***************************************************
 
+// tag::ios[]
+
+experimental::[]
+
+_coming soon_
```
Comment (bmorelli25) [inline docs/tab-widgets/distributed-trace-receive.asciidoc:40]:
```suggestion
_Not applicable._
```

**Pull request merged**